### ISSUE TITLE
"[Unexpected Error] -> KeyError -> 'cmsArticleWebOld'" is an error from akshare?

### ISSUE THREAD

**Issue by peppaonigri** [finanalyzer/openbb_akshare#7]
Hello Developer,

When I was using `open_akshare` and running your example:
```python
messages = obb.news.company('000001', provider='akshare')
```
I encountered the following error:
```
openbb_core.app.model.abstract.error.OpenBBError:
[Unexpected Error] -> KeyError -> 'cmsArticleWebOld'
```
This issue appears to originate from `akshare`, as the same problem occurs when running the command in OpenBB directly.

Additionally, the following command:
```python
prices = obb.equity.price.historical(symbol='06823', start_date="2025-06-01", end_date="2025-06-10", provider="akshare")
```
results in another error:
```
openbb_core.app.model.abstract.error.OpenBBError:
[Unexpected Error] -> IndexError -> single positional indexer is out-of-bounds
```
Is this also an issue with `akshare`?

To rule out potential problems with my `OpenBB` or `openbb_akshare` installation, I tested these working commands:
```bash
quote --symbol 000002 --provider akshare
```
```python
obb.equity.price.quote(symbol='000002', provider='akshare')
```

Could you please check if these are known issues with `akshare`?
